src/main/java/au/org/ala/biocache/web/OccurrenceController.java

```diff
@@ -1511,8 +1511,8 @@ private Object getOccurrenceInformation(String uuid, String im, HttpServletReque
                         ua.setUserId(authService.getDisplayNameFor(userId));
                     }
                 }
-                sd.addField("dbUserAssertions", userAssertions);
-            }
+                sd.addField("dbUserAssertions", assertions);
```

Comment:
Is this `dbUserAssertions` field used ? Its not referenced elsewhere so I don't think it appears in any output